config.ts

```diff
@@ -1,6 +1,6 @@
 import { AppConfigurationClient } from "@azure/app-configuration"
 
-const appconfig = new AppConfigurationClient(process.env.AZURE_APP_CONFIG_CONNECTION_STRING)
+const appconfig = new AppConfigurationClient(process.env.AZURE_APP_CONFIG_CONNECTION_STRING!)
```

Comment:
this triggers a warning from our lint config (see https://typescript-eslint.io/rules/no-non-null-assertion)